services/src/northbound/src/main/java/org/openkilda/northbound/config/SwaggerConfig.java

```diff
@@ -38,7 +38,6 @@
     public Docket api() {
         return new Docket(DocumentationType.SWAGGER_2)
                 .apiInfo(apiInfo())
-                .enableUrlTemplating(true)
                 .select()
                 .apis(RequestHandlerSelectors.any())
                 .paths(Predicates.not(PathSelectors.regex("/error.*")))
```

Comment:
Why not use "docket.genericModelSubstitutes(Optional.class);" instead of adding "required = false" to each optional RequestParam?

Reply:
Because it is not works. I tried it and it didn't give expected result.